### DISCUSSION TITLE
Template injection in ejs

### DISCUSSION THREAD

**Discussion by WTPang** [quasarframework/quasar#13438] [CLI - PWA mode]
While I ran "npm init quasar", it prompted me there was a risk. I ran "npm audit" in my quasar project folder and got following message. How do I fix this issue?

  High            Template injection in ejs
  Package         ejs
  Patched in      >=3.1.7
  Dependency of   @quasar/app-webpack [dev]
  Path            @quasar/app-webpack > ouch > ejs
  More info       https://github.com/advisories/GHSA-phwq-j96m-2c2q
found 1 high severity vulnerability in 871 scanned packages
  1 vulnerability requires manual review. See the full report for details.

**Comment by MilosPaunovic** [comment]
You keep your Quasar deps up-to-date. When a solution is published in the dependencies, Quasar will follow with a patch version bump and it will be resolved.

**Reply by WTPang** [reply]
Thanks for the update. Hope Quasar would update ejs dependency to version 3.1.7 or higher soon.

**Comment by pdanpdan** [comment]
You know it wouldn't make any difference anyway, except if you want to dos yourself

**Comment by Hashen110** [comment]
#13424 
#13425
